Dapper/SqlMapper.cs

```diff
@@ -2221,7 +2225,13 @@ public static object SanitizeParameterValue(object value)
 
         private static IEnumerable<PropertyInfo> FilterParameters(IEnumerable<PropertyInfo> parameters, string sql)

```

Comment:
Would change this to a `List<PropertyInfo>` return so consume know it's more specific now.